std_msgs/Bool ask

---

geometry_msgs/PoseStamped tool_pose
